manifests/00-config-custresdef.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: cloudcredentials.operator.openshift.io
   annotations:
     include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
```

Comment:
Updating this through `make update-vendored-crds` was required to pass `make verify`

Reply:
here's a little background on this annotation, https://github.com/openshift/enhancements/blob/master/enhancements/single-node-developer-cluster-profile.md